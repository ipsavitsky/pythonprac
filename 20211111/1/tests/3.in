A, B = OrderedComplex(123413, 128734), OrderedComplexMul(123413, 128734)
print(A<B, B@B)