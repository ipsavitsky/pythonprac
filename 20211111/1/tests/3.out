False 31803211325.0
